### PR TITLE
Add Base Construction

### PR DESCRIPTION
**Pull request by CosmoBro**

### CODE REVIEW

**Comment by CosmoBro** [comment]
can someone tell me why it doesn't merge? to fix it

File: NetKAN/ContractConfigurator-BaseConstuction.netkan [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    ],
+	"install"       : [
+        {
+            "find": "BaseConstruction",
+            "install_to": "GameData/ContractPacks"
+        }
+    ],
+	"suggests"      : [
+        {
+            "name": "KAS",
```
Comment (techman83) [inline NetKAN/ContractConfigurator-BaseConstuction.netkan]: You've got an extraneous ',' after "KAS".

**Comment by dbent** [comment]
Thanks, closing in favor of #1745, which fixes the command bumps the spec to `v1.2` which is required to support subdirectories in an `install_to` property.

**Comment by CosmoBro** [comment]
thanks

**Pull request closed without merging**